add new blogpost about DRS (#463)
usegalaxy-eu/website: _posts/2020-07-19-drs_sars-cov-2-paper.md

---
site: freiburg
title: "The landscape of SARS-CoV-2 RNA modifications"
tags: [paper]
doi: "10.1101/2020.07.18.204362 "
supporters:
 - denbi
 - unifreiburg
 - crc992
 - galaxy-europe
 - biodaten
---

[Nanopores](https://nanoporetech.com){:target="_blank"} are an awesome technology! We already gained experiences with it in our [StreetScience](https://streetscience.community),
where we show citizens and school kids what DNA sequencing is, how they can sequence with a USB-like device (Minion) and how they

> can analyze the data with [Galaxy](https://usegalaxy.eu){:target="_blank"}. But Nanopores can also sequence RNAs, directly. Direct RNA sequencing (DRS)

Reviewer: ```suggestion
can analyze the data with [Galaxy](https://usegalaxy.eu){:target="_blank"}. But Nanopores can also sequence RNAs directly. Direct RNA Sequencing (DRS)
```

can detect RNA modification in high-throughput! 

Wouldn't it be interesting to study the RNA modifications of SARS-CoV-2 and see if this helps in understanding SARS viruses?

|  |    |    |
|:---:|:-----------:|:---:|
| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; |[![DRS RAW signal](/assets/media/drs_fig-5.png)](https://www.biorxiv.org/content/10.1101/2020.07.18.204362v1){:target="_blank"}| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; |

> |  |Direct RNA sequencing raw electrical signals of downsampled reads obtained from unmodified RNA (IVT, black), from samples generated for this study and from isolate from a published korean data set (Fr1-3 and Kr, red).|  |

Reviewer: ```suggestion
|  |Direct RNA sequencing raw electrical signals of downsampled reads obtained from unmodified RNA (IVT, black), from samples generated for this study, and from isolated samples of a published Korean data set (Fr1-3 and Kr, red).|  |
```

<br>

We assembled a team of experts from the __University Medical Center Freiburg__, with access to the SARS-CoV-2 samples, from the 
__Medical Faculty of the Goethe University Frankfurt__ and Bioinformaticians from the __Galaxy community__ to do exactly this! Study the
landscape of SARS-CoV-2 RNA modifications.

We provide the __first DRS data__ of SARS-CoV-2 in infected human lung epithelial cells, the first (to our knowledge) public available DRS from Europe,

> and for the first time replicates :)

Reviewer: ```suggestion
and for the first time, replicates :)
```

> Of course we also offer the data and workflows for you to crunch the data on your own.

Reviewer: ```suggestion
Of course, we also offer the data and workflows for you to crunch the data on your own.
```

Take it, analyze it, kill the virus!

* [https://covid19.galaxyproject.org/direct-rnaseq](https://covid19.galaxyproject.org/direct-rnaseq){:target="_blank"}
* [https://bit.ly/usegalaxy-eu-covid19-data](https://bit.ly/usegalaxy-eu-covid19-data){:target="_blank"}

|  |    |    |
|:---:|:-----------:|:---:|
| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | [![Modified RNA regions](/assets/media/drs_fig-2.png)](https://www.biorxiv.org/content/10.1101/2020.07.18.204362v1){:target="_blank"} | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; |

> |  |Detection of modified RNA bases in SARS-CoV-2 sgRNAs. a: Heatmaps of Nanocompore p-value scores for modified sites for the 3 sample replicates (Fr1-3) as compared to unmodified RNA data from Kim et al.. The genomic regions containing top-1% modification scores are marked in red. b: Heatmaps of the predicted fraction of modified bases using Tombo. The red marks show top-1% modified sites per sample that are common in at least two of the three samples.|  |

Reviewer: ```suggestion
|  |Detection of modified RNA bases in SARS-CoV-2 sgRNAs. a: Heatmaps of Nanocompore p-value scores for modified sites for the 3 sample replicates (Fr1-3) as compared to unmodified RNA data from Kim et al. The genomic regions containing top-1% modification scores are marked in red. b: Heatmaps of the predicted fraction of modified bases using Tombo. The red marks show top-1% modified sites per sample that are common in at least two of the three samples.|  |
```
not sure about this suggestion :)

<br>

> Apart from the paper itself, check out our collection of [training material](https://training.galaxyproject.org){:target="_blank"},

Reviewer: ```suggestion
Apart from the paper itself, you can check out our collection of [training materials](https://training.galaxyproject.org){:target="_blank"},
```
